### PR TITLE
http Request

### PR DESCRIPTION
**Pull request by Sukisywang**
able to handle basic GET and POST

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -19,27 +22,25 @@ function App() {
...
-    })
-    .catch(function (error) {
-      console.log(JSON.stringify(error));
-    });
-  
-    if (question != null) {
-      document.getElementById("post").innerHTML =
-        "good question, let me think...";
-    }
+    axios
```
Comment (Kelvinyu1117) [inline client/src/App.jsx]: Another way to write is using the `async`/`await` syntax similar to above `GET` request.

File: server/app.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 @app.route('/GET', methods=['GET'])
```
Comment (Kelvinyu1117) [inline server/app.py]: Usually, we want to name the REST API endpoint to be a noun, like
* `/users`
* `/chat_history`

See: https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/

File: client/src/App.jsx [hunk 3]
```diff
@@ -19,27 +22,25 @@ function App() {
...
+    getUser();
   }
 
   return (
     <>
       <div className="card">
-        <button onClick={getUser}>Try GET</button>
-        <button onClick={myFunction}>Try POST</button>
+        {/*<button onClick={getUser}>Try GET</button>*/}
+        <button onClick={myFunction}>Click Me</button>
```
Comment (Kelvinyu1117) [inline client/src/App.jsx]: Minor: suggest to better descriptive name instead of `myFunction`.

File: note.txt [hunk 4]
```diff
@@ -0,0 +1,64 @@
+switch branch: git checkout fix/kelvin-setup
```
Comment (Kelvinyu1117) [inline note.txt:1]: Actually, you should checkout the `main` branch, and create your own feature branch.

File: note.txt [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+
+(base) C:\Users\User>conda info --envs
+# conda environments:
+#
+base                  *  C:\Users\User\miniconda3
+condaEnv1                C:\Users\User\miniconda3\envs\condaEnv1
+
+
+install nodejs:
```
Comment (Kelvinyu1117) [inline note.txt:35]: Once you install `npm` and `node.js`, you can simply just run `npm i` under the `client` folder, this command will pull all the libraries needed by this project.

**Review by Kelvinyu1117 — APPROVED** [review]
LGTM

File: client/src/App.jsx [hunk 6]
```diff
@@ -1,35 +1,49 @@
...
+      console.log(JSON.stringify(response));
+      document.getElementById("output").innerHTML = JSON.stringify(
+        response.data
+      );
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  function chatBox() {
```
Comment (Kelvinyu1117) [inline client/src/App.jsx]: need to be `async function chatBox`, and `await` all the async operations

**Comment by Kelvinyu1117** [comment]
I'm not sure how you install `axsio`, but normally you want to just do `npm -i axsio` instead of install it globally.
After you run this command, you should see `package.json` is updated.

**Comment by Sukisywang** [comment]
> I'm not sure how you install `axsio`, but normally you want to just do `npm -i axsio` instead of install it globally. After you run this command, you should see `package.json` is updated.

`npm -i axsio` is not working, i used npm install

**Comment by Kelvinyu1117** [comment]
> > I'm not sure how you install `axsio`, but normally you want to just do `npm -i axsio` instead of install it globally. After you run this command, you should see `package.json` is updated.
> 
> `npm -i axsio` is not working, i used npm install

My bad, you are correct.

**Comment by Kelvinyu1117** [comment]
Closed it as we decided to take @wanyeung's changes.

**Pull request closed without merging**